java-collections/src/main/java/dev/pawel/MyArrayList.java

```diff
@@ -15,6 +15,13 @@ private MyArrayList(int capacity, int size) {
         this.array = new Object[capacity];
     }
 
+    public MyArrayList(MyList<E> myList) {
+        this(DEFAULT_CAPACITY, DEFAULT_SIZE);
```

Comment:
I think it's better to initialize MyArrayList with the capacity based on the size of MyList instead of the default value to avoid resizing